examples/alert_stream_integration_endpoint/java_console_consumer/README.md

```diff
@@ -7,7 +7,7 @@ The Java consumer relies upon a configuration file which holds the username and
 Make a file called 'consumer.properties' and fill it with this:
 
 ```
-security.protocol=SASL_SSL
+security.protocol=PLAINTEXT
```

Comment:
why is this PLAINTEXT when it was SASL_PLAINTEXT above?

Reply:
I miss-typed that one, it should still be SASL_PLAINTEXT